### DISCUSSION TITLE
Visual Studio Copilot Hotkey is gone

### DISCUSSION THREAD

**Discussion by internetfx** [community/community#126966] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Now that GitHub Copilot is integrated into Visual Studio 2022, it doesn't seem to have a hotkey for forcing a suggestion, which was previously CTRL+ALT+\ in the editor. Ask Copilot and Open Copilot hotkeys still exist and are configurable. Can we please have the ability to add a hotkey for forcing a suggestion? Or am I missing something?

Thanks!

**Comment by ayim** [comment, marked as answer]
Hey folks, there's a docs update pending.
This functionality is still in Visual Studio, the shortcut has changed slightly.

You can use either alt + , or alt + , to trigger an inline suggestion if there isn't one already
These map to the Edit.NextSuggestion and Edit.PreviousSuggestion commands

**Reply by internetfx** [reply]
Thanks, that worked nicely.

Is the Copilot suggestions window removed as well? Will you have a document somewhere describing all these changes or is that coming?
